terraform/core/main.tf

```diff
@@ -90,9 +91,9 @@ resource "google_storage_bucket_object" "app-server-archive" {
   bucket = google_storage_bucket.fmltc-gae-source.name
 }
 
-resource "google_cloudfunctions_function" "frame-extraction" {
+resource "google_cloudfunctions_function" "perform-action" {
```

Comment:
@cmacfarl Here I changed frame-extraction to perform-action because the perform_action cloud function is used for many things, not just frame extraction.

Reply:
ok